### ISSUE TITLE
[feature request] Graph for speed and flow

### ISSUE THREAD

**Issue by tippfehlr** [mainsail-crew/mainsail#1616]
### Requested feature:

optionally display the speed and flow data in a chart.

Quick mockup I did:
![Screenshot_20231020_114314](https://github.com/mainsail-crew/mainsail/assets/46620049/4fe50d9b-10e0-42b0-b92e-492c55da0b6f)

just a little expandable section

### Solves the following problem:

Could help with seeing to which speeds the printer really accelerates with high speed and locked acceleration.

**Comment by meteyou** [comment]
Moonraker has no store for these values. So every time, you refresh the tab or close/open mainsail, the chart would start from 0.

**Comment by FoldZero** [comment]
Fliud now has 'Diagnostics' which is basicly this.

**Comment by meteyou** [comment]
@FoldZero yes, but this start every times you open webinterface again. as long, as there is no cache in moonraker to store it, we will not implement it.
